### ISSUE TITLE
Browser boot-looping

### ISSUE THREAD

**Issue by vlourme** [Ulyssedev/Rust-undetected-chromedriver#1]
Hello,

Using the example code provided in the README.md, the browser starts and stop directly, and it loops doing the same thing.
I don't have many logs, only this:

```
ChromeDriver already exists!
Detected patched chromedriver executable!
Starting chromedriver...
Starting ChromeDriver 114.0.5735.90 (386bc09e8f4f2e025eddae123f36f6263096ae49-refs/branch-heads/5735@{#1052}) on port 3949
Only local connections are allowed.
Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping ChromeDriver safe.
ChromeDriver was started successfully.
```

Might it be the patch not working on the automatically downloaded Chrome version?

Thank you

**Comment by vlourme** [comment]
I solved the issue.
The library download the latest ChromeDriver (for Chrome 114) but Chrome might not have the same version (in my case I had latest, v116), so it was an instant crash and restart loop.

I would avoid this would by downloading a Chromium client according to the ChromeDriver version in the project files, if the underlying library allows setting Chrome path.

**Comment by Ulyssedev** [comment]
Thanks for sharing your solution, a fix will be rolled soon on the next version.

**Comment by Ulyssedev** [comment]
Hey,

The issue should be fixed in v0.1.1 with cb460553d2cca75719fcce777c707211fc00070c. Chrome introduced a completely new endpoint to fetch the new chromedriver, more info [here](https://github.com/GoogleChromeLabs/chrome-for-testing#json-api-endpoints).

Feel free to open the issue again if it persists on macOS.

**Comment by vlourme** [comment]
Great! Thanks :)
